BrainPortal/app/views/nh_users/_form.html.erb

```diff
@@ -67,8 +67,10 @@
   </fieldset>
 
   <fieldset>
-    <%= f.label(:orcid, "Orcid Id") %>
+    <%= f.label(:orcid, "ORCID iD") %>
     <%= text_field_tag "meta[orcid]", @user.meta['orcid'], :size => 30 %><br />
+    <p class="field_note"><b>Note:</b> The url prefix is not required just enter all the 16 last symbols of your ORCID iD with dashes between each group of four, for example, 1234-5678-9098-765X
```

Comment:
Text change

```
You can enter the full ORCID URI, whichs starts with <em>https://</em>, or just the numerical identifier, e.g. <em>1234-5678-9012-345X</em>
````

Check the em or strong tag work (maybe Candice's template cancels their style here)

Reply:
Also in the text_field_tag (previous unchanged line) we shoudl not invoke your orcid_canonize method to show the full URI in the text box

Reply:
@prioux I would say 'the numerical part' or numerical 'numerical identifier (with dashes)' - dashes are required

Reply:
Sure. Fine with me. As long as it's clear.